migrations/2_deploy_contracts.js

```diff
@@ -27,8 +28,11 @@ module.exports = function (deployer) {
 
     await deployer.deploy(ProducerContract, UserContract.address);
```

Comment:
Vi uns vídeos no YT e passei a entender esse contrato, mas ainda não entendi o porque tem o `await deployer.deploy` para os contratos de `ActivistContract` e `ProducerContract` repetidos duas vezes. É um erro ou tem uma razão para isso?

Reply:
Também não entendi essa parte. Não estava assim antes e foi alguma alteração de algum outro PR, e não desse. Me parece que está duplicado sem razão para isso.

Reply:
Dei uma olhada aqui e parece ter sido adicionado na issue #69.

Reply:
Como essa questão não é relacionada a esse PR em si, vou fechar essa conversa e seguimos em outro local a discussão desse ponto beleza? Assim que souber te aviso.

Reply:
Está duplicado mesmo! Pode remover quando for mexer na sua issue @pjvperes. Valeu